src/Compilers/CSharp/Portable/CSharpResources.Designer.cs

```diff
@@ -98,14026 +98,317 @@ internal class CSharpResources {
         }
         
         /// <summary>
-        ///   Looks up a localized string similar to Elements cannot be null..
```

Comment:
This file has like 13k deletes.  Is that really supposed to be happening?  Also, we may want to just have a Pr that is the designer resources cleanup.